tagstudio/src/core/ts_core.py

```diff
@@ -9,143 +9,6 @@
 
 from src.core.library import Entry, Library
 
```

Comment:
The MyPy check is failing due to missing imports in the `ts_core.py` file. Seems like they were accidentally missed when moving the constants out into their own file.
```suggestion
from src.core.constants import TS_FOLDER_NAME, TEXT_FIELDS

```